Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,25 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int myAge = 35;
```

Comment:
The whole point of the homework was that you need to read input from console and use it in your program. 

What you did- is hardcoded all the input.

Please redo the homework. Refer to Console.ReadLine(), int.Parse, float.Parse()

Reply:
My apologies, I misunderstood. 

I have corrected those issues.